src/golang/lib/collections/integration/constants.go

```diff
@@ -37,7 +38,7 @@ const (
 func ParseService(s string) (Service, error) {
 	svc := Service(s)
 	switch svc {
-	case Postgres, Snowflake, MySql, Redshift, MariaDb, SqlServer, BigQuery, GoogleSheets, Salesforce, S3, Athena, AqueductDemo, Github, Sqlite, Airflow, Kubernetes, GCS, Lambda, MongoDB, Conda:
+	case Postgres, Snowflake, MySql, Redshift, MariaDb, SqlServer, BigQuery, GoogleSheets, Salesforce, S3, Athena, AqueductDemo, Github, Sqlite, Airflow, Kubernetes, GCS, Lambda, MongoDB, Conda, Databricks:
```

Comment:
@saurav-c does newer version of go allow us to this casting better?

Reply:
If it's unclear, I think we can add it to https://www.notion.so/aqueducthq/WIP-Case-studies-for-go1-19-migration-ee1761b26e4049bd9784f65cf83a74c3 somewhere to do some research in future